Scripts/Source/minai_VR.psc

```diff
@@ -336,7 +336,8 @@ Event OnUpdate()
     Main.RegisterEvent(lineToSay)
   Else
     ; Prompt AIFF to comment on it.
-    main.RequestLLMResponse(lineToSay, "chatnf_vr_1", actorName)
+    ; main.RequestLLMResponse(lineToSay, "chatnf_vr_1", actorName)
+    sex.sexTalkCollision(akActor, lineToSay)
```

Comment:
I think it's better use same method to utilize same cooldowns for talking otherwise collision talks will be in parallel with other possible talks.

Reply:
Makes sense. There is a global cooldown in the main request response functions that everything goes through that prevents them from being parallelized, though.